erpnext/stock/doctype/quality_inspection/quality_inspection.py

```diff
@@ -123,4 +123,12 @@ def postprocess(source, doc):
 		}
 	}, target_doc, postprocess)
 
-	return doc
\ No newline at end of file
+	return doc
+
+@frappe.whitelist()
+def check_is_compliance_item(item_code):
+	count=frappe.db.count('Compliance Item',item_code)
```

Comment:
This doctype currently exists in the Bloomstack Core app, maybe we should move it to ERPNext instead